platforms/config_settings/BUILD

```diff
@@ -0,0 +1,57 @@
+load("//platforms/config_settings:recycle_groups.bzl", "BUILDBUDDY_RECYCLE_GROUPS")
```

Comment:
Not entirely sure whether `//platforms/config_settings` or `//platforms/configs` is better.